phantombuster/web-browser-extension
# refactor: Remove shift + click to start a new session on a tab

Refactor: Remove Shift + click to start a new session on a tab. Clean unused Popup.

## 📝 What is this PR
Remove the functionality to create a new session when the user shift + click on the "Connect to XXXX" button. 
Remove the unused popup code.

## 🧪 QA Process

To test the extension : 
- npm i
- npm run clean
- npm run build:dev
- On Google Chrome, open chrome://extensions/
- Enable the Developer mode
- Click on Load Unpacked
- Select the dev-build folder
- The extension should be activated in the browser (don't forget to disable the official extension)

### ✅ Positive testing (should succeed)

To test the update : 

**Test when the user is already connected to the service:**
- Use a phantom
- Click on the "Connect to XXXX" button to get the cookie
- The phantom should work

**Test when the user is not connected to the service:**
- Use a phantom
- Click on the "Connect to XXXX" button. You should have a new tab opened
- Log in
- You are redirected to the PhantomBuster page

**Test to click the "Connect to XXXX" with Shift pushed**
- Use a phantom
- Click on the "Connect to XXXX" button with Shift pushed
- Nothing should change

## 👌 To check

-   [ ] There's no regression
-   [ ] Coding guidelines are followed ([Contributing](https://github.com/phantombuster/engineering-docs/blob/master/contributing.md), [Testing](https://github.com/phantombuster/engineering-docs/blob/master/testing.md))
